src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useContext } from 'react';
+import { TodosContext } from '../TodosContext';
+import { TodosFilter } from '../TodosFilter/TodosFilter';
+
+export const Footer: React.FC = () => {
+  const {
+    todos, setTodos, activeTodos, completedTodos,
+  } = useContext(TodosContext);
+
+  const clearCompleted = () => {
+    setTodos(todos.filter(todo => !todo.completed));
+  };
+
+  if (todos.length) {
```

Comment:
use conditional rendering in the parent component, not here
`todos.length > 0 && <Component />`